### DISCUSSION TITLE
Summernote uploading images in filesystem (storage)

### DISCUSSION THREAD

**Discussion by IlyaBuldakov** [Laravel-Backpack/community-forum#1049] [General]
Hey. I'm using summernote and when I insert images they are saved to the database in base64 format. How can I replace them with a link, provided that the images will be saved to the file system? Do I need custom JS for this?

**Comment by karandatwani92** [comment]
Hi @IlyaBuldakov 

This is Summernote's default image-handling behavior.

To customize the summernote's image upload behavior, you need to set `onImageUpload` [callback](https://summernote.org/deep-dive/#onimageupload).

For general customization, `options` attribute is available, but to customize callbacks, you can:
1. Create your own summernote field from the current summernote field:
```bash
php artisan backpack:field mysummernote --from=summernote
```
2. Customize your new field(in this case JS part) and use it.
